trio/testing/_fake_net.py

```diff
@@ -51,7 +51,7 @@ def _localhost_ip_for(family: int) -> IPAddress:
     elif family == trio.socket.AF_INET6:
         return ipaddress.ip_address("::1")
     else:
-        assert False
+        raise AssertionError()
```

Comment:
Add an error message to the three `AssertionError`s in this file - something like "unknown IP Address family". Or a `NotImplementedError` instead.
Also `# pragma: no cover`.

Reply:
Changed in a5afed7 and added reason in e1fdebc8